### PR TITLE
chromatic: fix auto accept args

### PR DESCRIPTION
**Pull request by burmudar**
A bit counter intuitive but `--auto-accept-changes` requires a value to be given ... and if you don't give if even if you specify the flag the default value is false
## Test plan
Make a change in this PR and see that it is auto accepted

### CODE REVIEW

**Review by BolajiOlajide — APPROVED** [review]

File: dev/ci/run-chromatic.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ ARGS=("$@")
 if [[ ${BUILDKITE_RETRY_COUNT:-0} == 1 ]]; then
   # Chromatic fails with exit-code 1 if the commit stays the same and instructs one to add `--force-rebuild`
   # So we detect when a build is retried and then add the flag accordingly
-  ARGS=("${ARGS[@]}" "--force-rebuild")
+  ARGS=("${ARGS[@]}" "--force-rebuild true")
```
Comment (Strum355) [inline dev/ci/run-chromatic.sh:8]: Whats this 🤔

Comment (burmudar) [reply dev/ci/run-chromatic.sh:8]: Screenshot 2024-01-08 at 22 02 13

Comment (eseliger) [reply dev/ci/run-chromatic.sh:8]: Lel

**Review by eseliger — APPROVED** [review]
Don’t forget the remove me 🙂

**Pull request merged**